src/main/minio.js

```diff
@@ -1602,7 +1612,7 @@ export class Client {
         throw new TypeError(`response header ${header} should be of type "string"`)
       }
     })
-    return this.presignedUrl('GET', bucketName, objectName, expires, respHeaders, cb)
+    return this.presignedUrl('GET', bucketName, objectName, expires, respHeaders, requestDate, cb)
```

Comment:
@dustin-H,
When you filed the issue for this PR, #724, as expected, you've also asked for `requestDate` parameter to be added for `presignedPutObject` api, but your fix does not include it.

Reply:
@dustin-H,
Makes sense.
I'll be testing your change today (which is required before giving a LGTM) and then give it a go.

Reply:
@dustin-H,
Makes sense.

Today, I'll be testing the fix (which is supposed to be done before giving a LGTM), and then give it a go.